packages/p2p-v2/src/components/PaymentMethodForm/PaymentMethodForm.tsx

```diff
@@ -208,7 +205,6 @@ const PaymentMethodForm = ({ onAdd, onResetFormState, ...rest }: TPaymentMethodF
                     isModalOpen={true}
                     onConfirm={() => {
                         onResetFormState();
-                        setIsModalOpen(false);
```

Comment:
For `PaymentMethodErrorModal`, the `onResetFormState` will destroy the component, so there's no need to use the `isModalOpen` state to control the modal